eng/Packages.props

```diff
@@ -312,6 +309,12 @@
     <!-- fix of vulnerability in 6.0.0 coming via Microsoft.TeamFoundationServer.Client -->
     <PackageVersion Include="System.Security.Cryptography.Xml" Version="8.0.0" />
     <PackageVersion Include="System.Security.Cryptography.Pkcs" Version="8.0.0" />
+    <!--
+      vs-extension-testing
+    -->
+    <PackageVersion Include="Microsoft.Windows.CsWin32" Version="0.3.106" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing.XUnit" Version="$(MicrosoftCodeAnalysisTestingVersion)" />
```

Comment:
Pretty sure I just marked this package as deprecated.

Reply:
Just the generic [Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing](https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing/)